ui/src/app/devices/views/DeviceList/DeviceListTable/DeviceListTable.test.tsx

```diff
@@ -0,0 +1,69 @@
+import { mount } from "enzyme";
```

Comment:
It would be nice if this new test file could be written using RTL. It shouldn't require many functional changes to what you've written here, let me know if you need a hand.

Reply:
As per discussion in https://github.com/canonical-web-and-design/maas-ui/pull/3302 we'll wait a sec on writing RTL tests, but in the meantime I'll continue to write enzyme tests in a way that'll make the migration easier.